### DISCUSSION TITLE
Let us modify sensitivity by typing a number in a text field

### DISCUSSION THREAD

**Discussion by Mechanizen** [ppy/osu#21714] [Ideas]
So many games have this problem, I'm not sure why this is still a standard. 

Modifying mouse sensivity by using a sliding cursor is super anoying. If you slide it to have a bit too much sensitivity then you are living this awkward moment when you struggle to hit the cursor and find the right spot. It's also utterly complicate to get the right value you are accustomed to by sliding a cursor. 

Being able to change mouse sensitivity by just modifying a value in a text area is just so much easier. This way you can input new values with your keyboard and test sensitivity with your mouse. If you don't click on anything the text field remains selected so if your sensitivity isn't good you can rewrite the value with your keyboard instead of struggling to hit the sliding cursor with a sensitivity you're not used to.

**Comment by Flamiii** [comment]
FYI, if your cursor is hovered over the slider, you can use the Left and Right arrow keys to adjust your sensitivity instead of having to move your cursor.

**Reply by Mechanizen** [reply]
Didn't know and that is cool but I think a text field would be more intuitive

**Comment by peppy** [comment]
All slider controls should have text input in the future. This needs to be added at a component level so it is usable everywhere.
